flybywiresim/a32nx
# fix: update UI_MANUFACTURER and UI_MODEL in locPak files

## Summary of Changes

Updated `AIRCRAFT.UI_MANUFACTURER` and `AIRCRAFT.UI_MODEL` to "`FlyByWire Simulations`" and "`A320 (LEAP)`".

## Screenshots (if necessary)


**Image 1:**
![hangar_title](https://user-images.githubusercontent.com/58574351/113647849-e31ed000-968b-11eb-9515-28d7a5fff01e.png)

**Image 2:**
![aircraft_selector](https://user-images.githubusercontent.com/58574351/113648223-8d96f300-968c-11eb-8748-c31ce09bb2b3.png)

**Image 3:**
![facts_window_title](https://user-images.githubusercontent.com/58574351/113648289-ad2e1b80-968c-11eb-9240-1e1d4e82002f.png)


## References





## Additional context

My first PR :)


Discord username (if different from GitHub): `ExampleWasTaken#0886`

## Testing instructions

1. Check that the title in the hangar matches the marking on image 1.
2. Check that the aircraft selector matches the marking on image 2.
3. Check that the title of the the window marked on image 3 matches the marking.
4. Check that the aircraft loads in sim.


## How to download the PR for QA

Every new commit to this PR will cause a new A32NX artifact to be created, built, and uploaded.

1. Make sure you are signed in to GitHub
1. Click on the **Checks** tab on the PR
1. On the left side, click on the bottom **PR** tab
1. Click on the **A32NX** download link at the bottom of the page
